components/resources/commands.grdp

```diff
@@ -489,4 +489,33 @@
   <message name="IDS_SHORTCUTS_PAGE_ADD_SHORTCUT" desc="Text for the button to add a new shortcut">
     Add
   </message>
+
+  <if expr="use_titlecase">
+    <message name="IDS_IDC_NEW_SPLIT_VIEW" desc="The label of the 'New Split View' tab context menu item">
+      New Split View
+    </message>
+    <message name="IDS_IDC_CLOSE_SPLIT_VIEW" desc="The label of the 'Close Split View' tab context menu item">
+      Close Split View
+    </message>
+    <message name="IDS_IDC_TILE_TABS" desc="The label of the 'Open in Split View' tab context menu item">
+      Open in Split View
+    </message>
+    <message name="IDS_IDC_BREAK_TILE" desc="The label of the 'Break into tabs' tab context menu item">
+      Break into Tabs
+    </message>
+  </if>
+  <if expr="not use_titlecase">
+    <message name="IDS_IDC_NEW_SPLIT_VIEW" desc="The label of the 'New Split View' tab context menu item">
+      New split view
+    </message>
+    <message name="IDS_IDC_CLOSE_SPLIT_VIEW" desc="The label of the 'Close Split View' tab context menu item">
+      Close split view
+    </message>
+    <message name="IDS_IDC_TILE_TABS" desc="The label of the 'Open in split view' tab context menu item">
+      Open in split view
+    </message>
+    <message name="IDS_IDC_BREAK_TILE" desc="The label of the 'Break into tabs' tab context menu item">
+      Break into tabs
```

Comment:
What will this do? Pretty sure the localizers will want to have more context.

Reply:
Yeah, good point. It exits the split view and break the bounded state between two tabs.

Updated desc.